chain/events/filter/index.go

```diff
@@ -496,12 +533,10 @@ func (ei *EventIndex) prefillFilter(ctx context.Context, f *eventFilter, exclude
 	}
 
 	if len(f.addresses) > 0 {
-		subclauses := make([]string, 0, len(f.addresses))
 		for _, addr := range f.addresses {
-			subclauses = append(subclauses, "emitter_addr=?")
 			values = append(values, addr.Bytes())
 		}
-		clauses = append(clauses, "("+strings.Join(subclauses, " OR ")+")")
+		clauses = append(clauses, "event.emitter_addr IN ("+strings.Repeat("?,", len(f.addresses)-1)+"?)")
```

Comment:
Nice fix for an off by one 👍

Reply:
This is just a cosmetic change, right ? Or did you see a diff with using `IN []` instead of `OR`

Reply:
no observed difference, but I haven't tried with a long list; I would hope the db engine would be able to see the same thing here but who knows without exhaustive testing!